examples/experimental/supply_and_demand/market_deep_dive_analysis.ipynb

```diff
@@ -0,0 +1,2213 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "<h1>Welcome to the Lab 🥼🧪</h1>\n",
+    "</center>\n",
+    "\n",
+    "## Deep dive analysis into a market\n",
+    "\n",
+    "In this notebook we will have a deep dive into the market dynamics of a specific market. We will explore topics such as:\n",
+    "* Changes in demand and supply\n",
+    "* Share of total inventory with price cuts\n",
+    "* Location of the inventory and prevalence of price cuts\n",
+    "* Supply of inventory coming from new construction\n",
+    "\n",
+    "#### What will you create in this notebook?\n",
+    "\n",
+    "##### Understand Changes in Supply and Demand YoY\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/changes_supply_yoy_Denver, CO.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "##### Determine Share of Inventory With Price Cuts\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/pct_inventory_price_reductions_line_chart_Denver, CO.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "##### Visualize Location of Available Inventory\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/map_active_listings_Denver, CO.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "##### Visualize Price Cutting Activity\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/map_active_listings_Denver, CO_with_price_reductions.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "##### Understand available inventory by Property Type\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/share_intentory_property_type_Denver, CO.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "##### Determine Price Cuts by Property Type\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"../../../images/price_cuts_biweekly_intentory_property_type_Denver, CO.png\" alt=\"Alt text\">\n",
+    "</p>\n",
+    "\n",
+    "#### Need help getting started?\n",
+    "\n",
+    "As a reminder, you can get your Parcl Labs API key [here](https://dashboard.parcllabs.com/signup) to follow along.\n",
+    "\n",
+    "To run this immediately, you can use Google Colab. Remember, you must set your `PARCL_LABS_API_KEY`.\n",
+    "\n",
+    "Run in collab --> [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ParclLabs/parcllabs-cookbook/blob/main/examples/experimental/supply_and_demand/markets_that_could_disrupt.ipynb)"
+   ]
```

Comment:
Please update this to the correct path for this notebook. I believe all thats required to change is the notebook name itself --> `market_deep_dive_analysis.ipynb`